commands/tempmute.js

```diff
@@ -0,0 +1,109 @@
+
+
+
+module.exports = {
+  name: 'tempmute',
+  description: 'A command similar to `mute`, but the effects being temporary.',
+  permissionError: 'You do not have permission to temporarily mute that member. (or there was an error doing so)',
+  permissions: ['KICK_MEMBERS', 'ADMINISTRATOR'],
+  minArgs: 2,
+  //maxArgs: 2
```

Comment:
we don't need max arguments (because if we set it to a certain amount, the command handler will just read it as a syntax error, which will be changed in the future, perhaps). the command handler as of 10/26/2020 handles the maxargs being null